app/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+
+app.use(express.static(path.join(__dirname, '..', 'dist')));
+
+app.get('/preferences.js', function (req, res) {
+  api_url = process.env.REACT_APP_API_URL
+  login_page = process.env.REACT_APP_LOGIN_PAGE_URL
+  logout_page = process.env.REACT_APP_LOGOUT_PAGE_URL
+  config = `var condaStoreConfig = {
+    REACT_APP_AUTH_METHOD: "cookie",
+    REACT_APP_AUTH_TOKEN: "",
+    REACT_APP_STYLE_TYPE: "green-accent",
```

Comment:
I think it's better to set defaults in one place and that place should be in the React app code, what do you think?